README.md

```diff
@@ -145,17 +145,17 @@ Filter Flags:
-      --gpu-memory-total-max float        Maximum Number of GPUs' total memory in GiB (Example: 4) If --gpu-memory-total-min is not specified, the lower bound will be 0
-      --gpu-memory-total-min float        Minimum Number of GPUs' total memory in GiB (Example: 4) If --gpu-memory-total-max is not specified, the upper bound will be infinity
+      --gpu-memory-total string           Number of GPUs' total memory (Example: 4 GiB) (sets --gpu-memory-total-min and -max to the same value)
+      --gpu-memory-total-max string       Maximum Number of GPUs' total memory (Example: 4 GiB) If --gpu-memory-total-min is not specified, the lower bound will be 0
+      --gpu-memory-total-min string       Minimum Number of GPUs' total memory (Example: 4 GiB) If --gpu-memory-total-max is not specified, the upper bound will be infinity
   -g, --gpus int                          Total Number of GPUs (Example: 4) (sets --gpus-min and -max to the same value)
       --gpus-max int                      Maximum Total Number of GPUs (Example: 4) If --gpus-min is not specified, the lower bound will be 0
       --gpus-min int                      Minimum Total Number of GPUs (Example: 4) If --gpus-max is not specified, the upper bound will be infinity
       --hibernation-support               Hibernation supported
       --hypervisor string                 Hypervisor: [xen or nitro]
-  -m, --memory float                      Amount of Memory available in GiB (Example: 4) (sets --memory-min and -max to the same value)
-      --memory-max float                  Maximum Amount of Memory available in GiB (Example: 4) If --memory-min is not specified, the lower bound will be 0
-      --memory-min float                  Minimum Amount of Memory available in GiB (Example: 4) If --memory-max is not specified, the upper bound will be infinity
+  -m, --memory string                     Amount of Memory available (Example: 4 GiB) (sets --memory-min and -max to the same value)
+      --memory-max string                 Maximum Amount of Memory available (Example: 4 GiB) If --memory-min is not specified, the lower bound will be 0
+      --memory-min string                 Minimum Amount of Memory available (Example: 4 GiB) If --memory-max is not specified, the upper bound will be infinity
       --network-interfaces int            Number of network interfaces (ENIs) that can be attached to the instance (sets --network-interfaces-min and -max to the same value)
       --network-interfaces-max int        Maximum Number of network interfaces (ENIs) that can be attached to the instance If --network-interfaces-min is not specified, the lower bound will be 0
       --network-interfaces-min int        Minimum Number of network interfaces (ENIs) that can be attached to the instance If --network-interfaces-max is not specified, the upper bound will be infinity
```

Comment:
Only having context from `ByteQuantity` code: 

> If no unit is appended, GiB is assumed

do you want that to be the common theme throughout? If so, then `vcpus-to-memory-ratio` description + logic may need to be updated

Reply:
great catch!! This was wrong before too, it was always using GiB for ratio